R/data-combined.R

```diff
@@ -14,6 +14,14 @@
 #' @import tidyr
 #' @import ospsuite.utils
 #'
+#' @param groups A string or a list of strings assigning the data set to a
```

Comment:
This is a common parameter across both `$add*()`. So, to avoid duplication of documentation, I am documenting it at the class documentation level.